libraries/ms-drivers/inc/adt_7470_fan_controller_defs.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+// ADT4740 Configuration and control commands
+// Datasheet: https://www.analog.com/media/en/technical-documentation/data-sheets/ADT7470.pdf
+
+#define ADT7470_CONFIG_REGISTER_1 \
+  0x40  // This register contains the STRT bit, Bit 0, which begins the monitoring cycle on the
+        // ADT7470.
+#define ADT7470_CONFIG_REGISTER_2 \
+  0x74  // Writing a 1 to Bit 0 in this register puts the ADT7470 in shutdown mode,
+        // which puts the part into a low current onsumption mode
+
+#define ADT7470_INTERRUPT_STATUS_REGISTER_1 0x41
+#define ADT7470_INTERRUPT_STATUS_REGISTER_2 0x42
+#define ADT7470_FAN_PULSES_PER_REV_REGISTER 0x43
+
+#define ADT7470_PWM1_DUTY_CYCLE 0x32
+#define ADT7470_PWM2_DUTY_CYCLE 0x33
+#define ADT7470_PWM3_DUTY_CYCLE 0x34
+#define ADT7470_PWM4_DUTY_CYCLE 0x35
+
+#define ADT7470_FAN_MODE_REGISTER_1 0x68  // bits 6 and 7 contorl the mode for PWM2 and PWM1
+#define ADT7470_FAN_MODE_REGISTER_2 0x69  // bits 6 and 7 contorl the mode for PWM4 and PWM3
+
+#define ADT7470_MANUAL_MODE_MASK 00001111
```

Comment:
I think you should define these with the `0b` prefix so they're actually the correct mask